terraform/app/modules/paas/main.tf

```diff
@@ -25,6 +25,52 @@ resource "cloudfoundry_user_provided_service" "papertrail" {
   syslog_drain_url = var.papertrail_url
 }
 
+resource "aws_s3_bucket" "documents_s3_bucket" {
+  bucket = local.documents_s3_bucket_name
+}
+
+resource "aws_s3_bucket_public_access_block" "documents_s3_bucket_block" {
+  bucket = aws_s3_bucket.documents_s3_bucket.id
+
+  block_public_acls       = true
+  block_public_policy     = true
+  ignore_public_acls      = true
+  restrict_public_buckets = true
+}
+
+resource "aws_iam_policy" "documents_s3_bucket_policy" {
+  name   = "${local.documents_s3_bucket_name}-policy"
+  path   = "/attachment_buckets_policies/"
+  policy = data.aws_iam_policy_document.documents_s3_bucket_policy_document.json
+}
+
+data "aws_iam_policy_document" "documents_s3_bucket_policy_document" {
+  statement {
+    actions   = ["s3:ListAllMyBuckets"]
+    resources = ["arn:aws:s3:::*"]
+    effect    = "Allow"
+  }
+  statement {
```

Comment:
I undertstand this is for the app top read and write objects inside the bucket. Then the resources should be: `"${aws_s3_bucket.documents_s3_bucket.arn}/*"`

Also, can we restrict it to:
```
        "s3:PutObject",
        "s3:GetObject",
        "s3:DeleteObject"
```